src/js/gutenberg.js

```diff
@@ -1,6 +1,10 @@
 import { debounce } from 'underscore';
```

Comment:
There are a few items in use in this file that are not listed in the dependencies (jQuery, apiRequest, domReady). Some of these are legacy issues but it would be good to add them to ensure this update doesn't end up changing the order in which files are included.

https://github.com/10up/insecure-content-warning/blob/42a8fbf51fd9ef62afb28c3070506143cd1b0a5b/includes/assets.php#L34-L40
:1234: To avoid a duplicate comment, are you able to review both JS files dependencies?

https://github.com/10up/insecure-content-warning/blob/42a8fbf51fd9ef62afb28c3070506143cd1b0a5b/includes/assets.php#L60-L66